server.py

```diff
@@ -297,7 +297,11 @@ def settings_page():
 
 @route('/system_info')
 def system_info():
-    viewlog = check_output(['sudo', 'systemctl', 'status', 'screenly-viewer.service', '-n', '20']).split('\n')
+    viewlog = None
+    try:
+        viewlog = check_output(['sudo', 'systemctl', 'status', 'screenly-viewer.service', '-n', '20']).split('\n')
```

Comment:
At some point, we should probably switch over to use the systemd python library to get logs instead.